contracts/lifebankcode/include/lifebankcode.hpp

```diff
@@ -195,6 +197,58 @@ CONTRACT lifebankcode : public contract
       string social_media_links,
       string photos);
 
+  /**
+   *
+   *  Saves the info related with an offer
+   *
+   * @param offer_name - Name of the offer
+   * @param sponsor_name - Name of the sponsor
+   * @param category - Category of the offer
+   * @param beginning_date - Available redemption offer start date
+   * @param ending_date - Available redemption offer end date
+   * @param cost - Offer cost
+   * @param description - Offer description
+   * @param restriction - Offer restriction
+   *
+   */
+  ACTION addoffer(
+      eosio::name offer_name,
```

Comment:
What if we use namespace eosio to avoid using eosio:: like code.
using namespace std;

Reply:
I had it like that, but I changed to follow that pattern, but I can revert it, please @JustinCast confirm me

Reply:
working on

Reply:
In my personal opinion I think the code is cleaner when you avoid using eosio:: or std:: like styles. But if the pattern is already implemented in the whole project. We can follow it. So it's on you.
@xavier506 what would you say?